### PR TITLE
Disable 'method_prefix' globally

### PR DESCRIPTION
**Pull request by jorge07**
Add support for https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1507

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -103,6 +103,7 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->scalarNode('default_format')->defaultNull()->end()
+                        ->scalarNode('method_prefix')->defaultTrue()->end()
```
Comment (xabbuh) [inline DependencyInjection/Configuration.php]: By just looking at the name of the option I would expect to be able to pass a string that will be the prefix. To have an option acting as a toggle to turn on/off the feature I would rather rename it to something like `prefix_methods`.

Comment (jorge07) [inline DependencyInjection/Configuration.php]: I just followed the same behavior as the existing option that appears in the doc http://symfony.com/doc/current/bundles/FOSRestBundle/7-manual-route-definition.html#method-name-prefix

But maybe we can add that option too

Comment (jorge07) [inline DependencyInjection/Configuration.php]: Ping @xabbuh . Should I change it?

Comment (xabbuh) [inline DependencyInjection/Configuration.php]: Yeah, I think naming that option `method_name_prefix` was not the best idea. Let's not make the same mistake here again. Can you please rename it? :)

Comment (jorge07) [inline DependencyInjection/Configuration.php]: Done

**Pull request merged**